core-web/nx.json

```diff
@@ -24,8 +24,8 @@
         }
     },
     "cli": {
-        "analytics": false,
-        "defaultCollection": "@nxext/stencil"
+        "defaultCollection": "@nrwl/angular",
+        "packageManager": "npm"
```

Comment:
Cool! I was wondering why the default collection was `@nxext/stencil`.

Reply:
yes.